data-sources/argocd/config.yml

```
@@ -0,0 +1,31 @@
+id: argocd
+
+name: ArgoCD 
```

I think this should be:
```suggestion
displayName: ArgoCD 
```

---

Whoops! fixed :)